g_appcode.go

```diff
@@ -17,15 +17,14 @@ package main
 import (
 	"database/sql"
 	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
```

Comment:
please move to the bottom. Actually I want to follow the goimport's style:

import{
    //system package

   // third package

  // project package
}